domains/certificates/Query.API/API.UnitTests/ValueObjects/UnixTimestampTest.cs

```diff
@@ -46,6 +46,20 @@ public void RoundToLatestHour()
         var now = DateTimeOffset.Now;
         var latestHour = new DateTimeOffset(now.Year, now.Month, now.Day, now.Hour, 0, 0, now.Offset);
         Assert.Equal(UnixTimestamp.Create(latestHour), UnixTimestamp.Create(now).RoundToLatestHour());
+
+        var alignedHour = new DateTimeOffset(2024, 2, 24, 12, 0, 0, TimeSpan.Zero);
+        Assert.Equal(UnixTimestamp.Create(alignedHour), UnixTimestamp.Create(alignedHour).RoundToLatestHour());
+    }
+
+    [Fact]
+    public void RoundToNextHour()
```

Comment:
given when then?  :)

Reply:
It contains multiple examples, so renamed to _RoundToNextHourExamples_